cni-plugin/integration/tests/calico/calico_test.go

```diff
@@ -0,0 +1,26 @@
+package calico_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/linkerd/linkerd2-proxy-init/cni-plugin/integration/testutil"
+)
+
+const (
+	ConfigDirectory = "/host/etc/cni/net.d"
+	CalicoConflist  = "10-calico.conflist"
```

Comment:
Calico ships a `.conflist` and not a `.conf` That's great, that removes one of the Docker-based tests then.

Reply:
It seems to ship with a `.conflist`, at least the version we're using here.

```
:; docker exec -it k3d-calico-server-0 sh
/ # cd /etc/
/etc # ls
cni  hostname  hosts  mtab  nsswitch.conf  os-release  rancher  resolv.conf  ssl  strongswan
/etc # cd cni/net.d/
/etc/cni/net.d # ls
10-calico.conflist  calico-kubeconfig
/etc/cni/net.d #
```